### PR TITLE
Rename UNIT_TESTS to DISABLE_TESTS for travis integration

### PR DESCRIPTION
**Pull request by haudren**
In addition, the previous UNIT_TESTS was not used, so a IF() macro was added.

### CODE REVIEW

File: tests/CMakeLists.txt [hunk 1]
```diff
@@ -25,7 +25,7 @@ include_directories(${Boost_INCLUDE_DIRS})
 set(HEADERS arms.h)
 
 macro(addUnitTest name)
-  if(${UNIT_TESTS})
+  if(NOT ${DISABLE_TESTS})
```
Comment (gergondet) [inline tests/CMakeLists.txt]: This (and the related `endif` obviously) can be removed as the file will not be included if `DISABLE_TESTS` is ON.

**Comment by gergondet** [comment]
Other than this comment this should be ok to merge.

**Comment by gergondet** [comment]
I've taken the liberty to rebase this PR on master. I'll merge once the CI passes.

**Comment by haudren** [comment]
Thanks :smile_cat:

**Pull request merged**